sdm-api-app.groovy

```diff
@@ -435,6 +435,8 @@ def processCameraTraits(device, details) {
     def imgRes = details?.traits?.get('sdm.devices.traits.CameraImage')?.maxImageResolution
     imgRes?.width ? sendEvent(device, [name: 'imgWidth', value: imgRes.width]) : null
     imgRes?.height ? sendEvent(device, [name: 'imgHeight', value: imgRes.height]) : null   
+    def room = details?.parentRelations?.getAt(0)?.displayName
```

Comment:
I think this is a lingering merge conflict that ended up as a duplicate? covered in `def processTraits`, lines 367-368

Reply:
Agreed, my error.